edgelet/iotedge/src/check/checks/check_agent_image.rs

```diff
@@ -0,0 +1,226 @@
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.0.9.5-linux-arm64v8",
+                0,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.0.9.5-windows-amd64",
+                0,
+            ),
+            ("mcr.microsoft.com/azureiotedge-agent:1.1", 0),
+            ("mcr.microsoft.com/azureiotedge-agent:1.1.0", 0),
+            ("mcr.microsoft.com/azureiotedge-agent:1.1.0-linux-amd64", 0),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.1.0-linux-arm32v7",
+                0,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.1.0-linux-arm64v8",
+                0,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.1.0-windows-amd64",
+                0,
+            ),
+            ("mcr.microsoft.com/azureiotedge-agent:1.2.0-rc1", 1),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc1-linux-amd64",
+                1,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc1-linux-arm32v7",
+                1,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc1-linux-arm64v8",
+                1,
+            ),
+            ("mcr.microsoft.com/azureiotedge-agent:1.2.0-rc2", 1),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc2-linux-amd64",
+                1,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc2-linux-arm32v7",
+                1,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc2-linux-arm64v8",
+                1,
+            ),
+            ("mcr.microsoft.com/azureiotedge-agent:1.2.0-rc3", 1),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc3-linux-amd64",
+                1,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc3-linux-arm32v7",
+                1,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc3-linux-arm64v8",
+                1,
+            ),
+            ("mcr.microsoft.com/azureiotedge-agent:1.2.0-rc4", 1),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc4-linux-amd64",
+                1,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc4-linux-arm32v7",
+                1,
+            ),
+            (
+                "mcr.microsoft.com/azureiotedge-agent:1.2.0-rc4-linux-arm64v8",
+                1,
+            ),
+            ("$upstream:4443/azureiotedge-agent:3.0.9.5-linux-arm32v7", 1),
+            ("randomImage/randomImage:1.0", 1),
+        ];
+
+        for agent_image in &array {
+            let result = if let Ok(CheckResult::Ok) = check_agent_image_version(&agent_image.0) {
+                1
```

Comment:
Use `true` and `false` instead of `1` and `0`

Reply:
thanks, old C programmer habit. Rust has true  and false :).